Hunting Queries/Microsoft 365 Defender/General queries/Endpoint Linux Agent Health Status Report

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Please have an **Id** field here: 
Id is just a standard GUID. You can generate from just about any development tool, online GUID generator, or from PowerShell via the [New-GUID cmdlet](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/new-guid?view=powershell-6).